test/selenium/print_test.js

```diff
@@ -0,0 +1,14 @@
+// 03 Print test using browserstack
+
+var webdriver = require('browserstack-webdriver');
+
+var runTest = function(cap, driver, target) {
+  // Click on "Drucken"
+  driver.findElement(webdriver.By.xpath("//a[@id='printHeading']")).click();
+  // Wait 1 sec for the menu to pull down
+  driver.sleep(1000);
```

Comment:
Maybe you can also check other attributes of the button directly. Maybe the 'visibility' attribute of the button is changed after the animation. You can inspect these in the developer tools in the browser.